python/cudf/cudf/utils/hash_vocab_utils.py

```diff
@@ -245,7 +245,7 @@ def hash_vocab(
     """
     Write the vocab vocabulary hashtable to the output_path
     """
-    np.random.seed(1243342)
```

Comment:
@VibhuJawa Switching to `default_rng` changes the random values generated and thus updates to `python/cudf/cudf/tests/data/subword_tokenizer_data/bert_base_cased_sampled/vocab-hash.txt` have been made, will that be an issue for tokenizer?

Reply:
I dont think it should be , will have to test with some common vocabulary files to be sure